eZ/Bundle/EzPublishCoreBundle/Command/CleanupVersionsCommand.php

```diff
@@ -221,6 +258,16 @@ protected function getObjectsIds($keep, $status)
             $query->setParameter('status', $this->mapStatusToVersionInfoStatus(self::VERSION_PUBLISHED));
         }
 
+        if ($excludedContentTypeIds) {
+            $expr = $query->expr();
+            $query->andWhere(
+                $expr->notIn(
+                    'c.contentclass_id',
+                    $excludedContentTypeIds
```

Comment:
Please use parameters e.g.
```php
            $query
                ->andWhere(
                    $expr->notIn(
                        'c.contentclass_id',
                        ':contentTypeIds'
                    )
                )
                ->setParameter('contentTypeIds', Connection::PARAM_INT_ARRAY);
```

Otherwise I can perform here an SQL injection by executing:
```
php ./app/console ezplatform:content:cleanup-versions --keep=5 --excluded-content-types="1)) OR 5=? --"
```
It's probably minor since it is console, but we never know how end-developers would hook up input and there's an easy way to sanitize it.

Reply:
Oh... you got me, great catch. My bad 🤦‍♂Fixed in: https://github.com/ezsystems/ezpublish-kernel/pull/2691/commits/ace6d82b6a00d9338d64ab4fe29b6c798a1e8bac